k8s-demo.py

```diff
@@ -297,15 +300,23 @@ def show_wordpress_info():
     print("URL: https://{}".format(data["master_ip"]))
 
 
-def run_shell_command(commands: list, print_log: bool = VERBOSE_MODE) -> str:
+def run_shell_command(commands: list, print_log: bool = VERBOSE_MODE, retries: int = 0) -> str:
     proc = subprocess.Popen(commands, stdout=subprocess.PIPE, shell=True)
     (out, err) = proc.communicate()
     result = out.decode('UTF-8')
+    if err:
+        print(bcolors.FAIL + "Error executing: {}".format(*commands) + bcolors.ENDC)
+        print(bcolors.FAIL + "{}".format(err.decode('UTF-8')) + bcolors.ENDC)
+        if retries < MAX_RETRIES:
+            print(bcolors.HEADER + "Retrying ({}/{})".format(retries, MAX_RETRIES) + bcolors.ENDC)
+            sleep(0.2)
+            retries = retries + 1
+            return run_shell_command(commands, print_log, retries)
```

Comment:
I gave similar feedback in another MR. You could consider the retrying module https://pypi.org/project/retrying/ instead of recursion.